core/src/main/java/org/apache/iceberg/io/FileWriter.java

```diff
@@ -54,6 +54,20 @@ default void write(Iterable<T> rows) {
    */
   void write(T row);
 
+  /**
+   * Returns the file path that are currently opened.
```

Comment:
```suggestion
   * Returns the file path that is currently opened.
```
Is "opened" important here?  what happens if the writer is closed?